packages/babel-compat-data/scripts/data/plugin-features.js

```diff
@@ -99,6 +99,7 @@ const es2017 = {
 
 const es2018 = {
   "proposal-async-generator-functions": "Asynchronous Iterators",
+  "proposal-numeric-separator": "numeric separator",
```

Comment:
It is not part of es2018. Could you create a new "shippedProposal" object at the end of the file?

And in the `Object.assign` call, it should come before "es2020".